bigflow/scaffold/templating.py

```diff
@@ -10,7 +10,7 @@
 logger = logging.getLogger(__name__)
 
 
-class SkipThisTemplate(BaseException):
+class SkipThisTemplate(Exception):
```

Comment:
Can't we use a standard library exception instead of `SkipThisTemplate`?

Reply:
This is a custom exception to prevent file from being rendered.
It is not an indicator of an error, but is more like "hacky" control-flow (like StopIteration).
We can't use any default error here as it catched by 'templating.py' module.